jellyfin-model/src/commonMain/kotlin/org/jellyfin/sdk/model/RegexValidator.kt

```diff
@@ -0,0 +1,10 @@
+package org.jellyfin.sdk.model
+
+
+public object RegexValidator {
```

Comment:
I don't want to introduce additional code generation that relies on existing classes. This is the main reason I didn't add the centralized regex validation myself yet.